Failed to load plugin 'prettier' declared in 'CLIOptions': Cannot find module 'eslint-plugin-prettier'

I reviewed older issues, but didn't find anything that matched this issue.

Prettier is no longer working properly in Atom. I have all my dependencies updated (as far as I can tell). I've tested against older versions of the code and this issue only occurs with these updated dependencies (list below).

Here is the error I get in Atom when I try to run auto-formatting with Prettier:
![https://i.imgur.com/wUMJsWG.jpg](https://i.imgur.com/wUMJsWG.jpg)

Here are the dev dependencies from my package.json file:
```
"eslint": "^6.0.1",
"eslint-config-prettier": "^4.3.0",
"eslint-plugin-node": "9.1.0",
"eslint-plugin-prettier": "^3.1.0",
"prettier": "^1.18.2",
```

I've rolled the dependencies above back to older versions and the error goes away, prettier functions as expected. 

Running prettier on the command line seems to work fine. It's just the Atom integration/plugin that is failing.

In my case, the `.eslintrc` is in the root of the project and all my plugins are installed locally. Also tried to set `root` as `true` but have the same error
```javascript
prettier-atom failed: Failed to load plugin 'emotion' declared in 'CLIOptions': Cannot find module 'eslint-plugin-emotion' 
```

> Can you guys read through here and see if any of this applies to you?

I have everything installed locally as well.

**package.json**
```json
"devDependencies": {
  "babel-eslint": "^10.0.2",
  "eslint": "^6.1.0",
  "eslint-plugin-prettier": "^3.1.0",
  "prettier": "^1.18.2",
  "prettier-eslint": "^9.0.0"
}
```

**.eslintrc**
```json
"parser": "babel-eslint",
"parserOptions": {
  "ecmaVersion": 6,
  "sourceType": "module",
  "ecmaFeatures": {
    "modules": true
  }
},
"plugins": ["prettier"],
"rules": {
  "prettier/prettier": "error"
}
```

**Error:**

> prettier-atom failed: Failed to load plugin 'prettier' declared in 'CLIOptions': Cannot find module 'eslint-plugin-prettier' Require stack: /placeholder.js

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.6.3",
+    "@lewisl9029/prettier-eslint": "^9.0.1",
     "atom-linter": "^10.0.0",
     "atom-package-deps": "^5.1.0",
     "atom-text-buffer-point": "^1.0.0",
@@ -32,7 +33,7 @@
     "lodash": "^4.17.13",
     "loophole": "^1.1.0",
     "prettier": "1.18.2",
-    "prettier-eslint": "^9.0.0",
+    "prettier-eslint": "npm:@lewisl9029/prettier-eslint",
     "prettier-stylelint": "^0.4.2",
     "read-pkg-up": "^6.0.0",
     "yarn-global-modules": "^1.0.1"

